package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "dallinger",
+  "version": "0.1.0",
+  "description": "Dallinger experiment server",
+  "dependencies": {
+  },
+  "devDependencies": {
+    "browser-sync": "^2.18.8",
+    "browser-sync-webpack-plugin": "^1.1.4",
+    "webpack": "^2.4.1",
+    "uglify-es": "^3.0.28"
+  },
+  "scripts": {
+    "build": "webpack",
+    "dev": "webpack --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Dallinger/Dallinger.git"
+  },
+  "keywords": [
+    "experiment",
+    "web",
+    "social",
+    "science"
+  ],
+  "author": "Jordan W. Suchow",
```

Comment:
Can you make this match the LICENSE file?

Reply:
You mean using `The Regents of the University of California`? I was just following the Griduniverse `package.json` here, but it's no problem to change.

Reply:
Ah, I guess it depends on whether `author` means copyright owner or maintainer in NPM packages?